components/emoji/emoji.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <span :class="emojiSize">
+  <span :class="['d-d-inline-block', 'd-icon', emojiSize]">
```

Comment:
We try not to use utility classes in dialtone-vue anymore due to their immutability. I know there are still tons of components in dialtone-vue using them, but any new stuff we put in should use defined css classes instead, even if it's super simple like this.

You can simply define a class like this in the <style> section:

```
.dt-emoji {
  display: inline-block;
}
```

Reply:
Fixed in b148e5fd291f98af59d7ca8c65417f79a0be2a6b
I still had to leave the classes `d-icon` and `emojiSize` since that class defines the variables for the sizes, I suppose thats ok?

Reply:
Yep, those ones are not utility classes